Makefile

```diff
@@ -312,6 +312,7 @@ docker.prebuilt:
 	cp ${GO_TOP}/bin/{istio_ca,node_agent} security/docker
 	cp docker/ca-certificates.tgz security/docker/
 	time (cd security/docker && docker build -t ${HUB}/istio-ca:${TAG} -f Dockerfile.istio-ca .)
+	time (cd security/docker && docker build -t ${HUB}/node-agent:${TAG} -f Dockerfile.node-agent .)
```

Comment:
There is a node-agent-test and an istio-ca-test image.. what are they meant for? if they are useless, can you remove them?

Reply:
They are used for integration with different input params.